gravitee-apim-portal-webui-next/src/components/button/button.scss

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@use 'sass:map';
+@import '../../scss/theme/theme';
+
+$button-primary-outline-color: map.get($mat-primary, main);
+
+button.mdc-button {
+  padding: 0.625rem 1.5rem;
+  --mdc-outlined-button-container-shape: 100px;
+  --mdc-filled-button-container-shape: 100px;
+  --mdc-outlined-button-label-text-tracking: 0.1px;
+  --mdc-filled-button-label-text-tracking: 0.1px
+}
+
+button.mat-mdc-outlined-button.mat-primary {
+  --mdc-outlined-button-outline-color: #{$button-primary-outline-color};
```

Comment:
Isn't the outline supposed to be `#79747E` (light gray) instead of the primary color?

Reply:
Done